src/test/java/dev/academy/patterns/singleton/ConfigHolderLazyTest.java

```diff
@@ -0,0 +1,14 @@
+package dev.academy.patterns.singleton;
+
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.*;
+
+public class ConfigHolderLazyTest {
+
+    @Test
+    public void getPoolSize() {
+        assertThat(ConfigHolder.getInstance().getPoolSize()).isEqualTo(0);
```

Comment:
Please use the class you actually want to test also here.
-> `ConfigHolderLazy` instead of `ConfigHolder`